lab_1/start.py

```diff
@@ -3,22 +3,52 @@
 """
 
 import os
+import main
 
 PATH_TO_LAB_FOLDER = os.path.dirname(os.path.abspath(__file__))
 PATH_TO_TEXTS_FOLDER = os.path.join(PATH_TO_LAB_FOLDER, 'texts')
+PATH_TO_PROFILES = os.path.join(PATH_TO_LAB_FOLDER, 'profiles')
 
 if __name__ == '__main__':
-
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'en.txt'), 'r', encoding='utf-8') as file_to_read:
         en_text = file_to_read.read()
 
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'de.txt'), 'r', encoding='utf-8') as file_to_read:
         de_text = file_to_read.read()
 
+    with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'la.txt'), 'r', encoding='utf-8') as file_to_read:
+        la_text = file_to_read.read()
+
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'unknown.txt'), 'r', encoding='utf-8') as \
             file_to_read:
         unknown_text = file_to_read.read()
 
+    TOP_N = 9
+    unknown_profile = main.create_language_profile('unk', unknown_text, [])
+
+    de_profile_1 = main.load_profile(os.path.join(PATH_TO_PROFILES, 'de.json'))
+    en_profile_1 = main.load_profile(os.path.join(PATH_TO_PROFILES, 'en.json'))
+    la_profile_1 = main.load_profile(os.path.join(PATH_TO_PROFILES, 'la.json'))
+    profiles_1 = [de_profile_1, en_profile_1, la_profile_1]
+
+    RESULT_1 = main.detect_language_advanced(unknown_profile, profiles_1, [], TOP_N)
+    print('External detection result:', RESULT_1)
+
+    de_profile_2 = main.create_language_profile('de', de_text, [])
+    en_profile_2 = main.create_language_profile('en', en_text, [])
+    la_profile_2 = main.create_language_profile('la', la_text, [])
+    profiles_2 = [de_profile_2, en_profile_2, la_profile_2]
+
+    RESULT_2 = main.detect_language_advanced(unknown_profile, profiles_2, [], TOP_N)
```

Comment:
can you describe in writing what happens in `start.py`?

Reply:
it detects a language twice: first, using profiles from json files that have already been made, and second, using texts with which new profiles are being made with the create_language_profile function.

Reply:
imports module os and main.py
using os, creates paths to folders
reads files with texts in different languages and lastly the unknown text
icreates an unknown language profile
then detects its language twice: first, using loaded profiles from json files that have already been made, and second, using texts with which new profiles are being made with the create_language_profile function
checks if both results are correct